src/main/java/frc/robot/subsystems/ball/intake/IntakeImpl.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems.ball.intake;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import ca.team3161.lib.robot.subsystem.RepeatingPooledSubsystem;
+import java.util.concurrent.TimeUnit;
+
+public class IntakeImpl extends RepeatingPooledSubsystem implements Intake {
+
+    Solenoid intakeSolenoid;
+    WPI_TalonSRX intakeMotorController;
+    boolean extended;
+    int intakeDirection;
+    double intakeSpeed;
+
+    public IntakeImpl() {
+        super(250, TimeUnit.MILLISECONDS); // Figure out actual value
+        this.intakeSolenoid = new Solenoid(frc.robot.RobotMap.INTAKE_SOLENOID_CHANNEL);
+        this.intakeMotorController = new WPI_TalonSRX(frc.robot.RobotMap.INTAKE_TALON_PORT);
+        this.extended = this.intakeSolenoid.get();
+        this.intakeSpeed = 0.8d;
+    }
+    
+    public void extend(int motorDirection) {
```

Comment:
Is this intended to be +-1, or is 0 also intended as an acceptable value? Will things work properly if I put in -5? What about Integer.MAX_VALUE? Does +1 mean roll inward or outward?

I'd probably define a MotorDirection enum or something for this purpose rather than using an int.